src/modules/user/user.service.ts

```diff
@@ -25,6 +18,24 @@ export default class UserService {
     private userRepository: Repository<User>
   ) {}
 
+  async findOne(user_id: string): Promise<User> {
+    const user = await this.userRepository.findOne({ where: { id: user_id } });
```

Comment:
Wrap your db call in a try catch block for easier debugging

Reply:
Done!

Reply:
Done!